src/main/java/com/exasol/adapter/dynamodb/documentnode/dynamodb/DynamodbObject.java

```diff
@@ -0,0 +1,46 @@
+package com.exasol.adapter.dynamodb.documentnode.dynamodb;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.exasol.adapter.dynamodb.documentnode.DocumentNode;
+import com.exasol.adapter.dynamodb.documentnode.DocumentObject;
+
+/**
+ * This class represents a DynamoDB map value.
+ */
+public class DynamodbObject implements DocumentObject<DynamodbNodeVisitor> {
```

Comment:
Sorry to pick on names today :) Here maybe you can also rename the class to `DynamodbObjectMap`?

Reply:
Similar for other, if they extend DocumentArray, then rename to `*List`.